array-map/main.js

```diff
@@ -0,0 +1,18 @@
+const numbers = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12];
+
+const languages = [
+  'Hypertext Markup Language',
+  'Cascading Style Sheets',
+  'ECMAScript',
+  'PHP Hypertext Preprocessor',
+  'Structured Query Language'
+];
+
+const doubled = numbers.map(number => number * 2);
+const prices = numbers.map(numbers => numbers.toFixed(2));
```

Comment:
`toFixed` is a good way to go here but you are missing the `$` to go with your prices